### ISSUE TITLE
How to increase video memory?

### ISSUE THREAD

**Issue by weeniemount** [foxlet/macOS-Simple-KVM#553]
Yesterday i installed macOS Catalina, and the peformance is good. But, its at 3MB of video memory, wich isnt very good. I would like to increase to something like 100 or 200mb, beacuse 3mb is low. I have also a libvirt xml file for the KVM.

**Comment by weeniemount** [comment]
Here are the specs. 
![Unnamed](https://user-images.githubusercontent.com/59868100/160395751-d3b036c7-cb74-4650-8347-a7e6f5cbe6e2.png)

**Comment by notAperson535** [comment]
@idkthegithub You must pass-through a GPU, but it can't be integrated graphics

**Comment by darkyelox** [comment]
@notAperson535 I disagree, I had successfully passthrough my Intel HD Graphics 510 of my laptop to monterey on Arch linux using [OSX_GVT_D](https://github.com/vivekmiyani/OSX_GVT-D) repository and compiling my own [i915ovmf rom](https://github.com/patmagauran/i915ovmfPkg/wiki/Using) (but first I used [OSX_KVM](https://github.com/kholia/OSX-KVM) repo for create and install my monterey and then copied the mac_hdd_ng.img to OSX_GVT_D after ensuring the mac os is installed successfully). Mac detects the graphic but in my case for some reason it is using only 7mb of video, do you know how to increase it in open core?

**important note**: my laptop is an asus optimus (iGPU + dGPU nvidia 920mx) so I used optimus-manager in arch to switch to 'integrated' card (no hybrid for the moment, I should test) and then unbind iGPU and framebuffer from host to guest using libvirt and then starts the OS using the scripts from OSX_GVT_D

Here is the proof 
![IMG_20220328_101428481](https://user-images.githubusercontent.com/8183191/160432526-d142d5d9-177b-4c15-8e75-8799dff9f9e7.jpg)

Other think isn't working is my touchpad (because for some reason QEMU complains when I use the EVDEV for my touchpad saying "is is not a evdev device" but maybe I could fix it using python-evdev or something else)

**Comment by notAperson535** [comment]
@darkyelox I think you can actually do it, but they don't recommend it. You can still do it though.

**Comment by darkyelox** [comment]
@notAperson535 yes, it is better to passthrough an AMD card but for those whom doesn't have one or (as my case) have an Nvidia that isn't supported by OSX is a good option to get at least a better performance from the machine.

**Comment by weeniemount** [comment]
> @idkthegithub You must pass-through a GPU, but it can't be integrated graphics

I have a AMD card and intergrated graphics. My AMD card isnt used much, and i would use the AMD card for the passthrough.

**Comment by notAperson535** [comment]
> > @idkthegithub You must pass-through a GPU, but it can't be integrated graphics
> 
> I have a AMD card and intergrated graphics. My AMD card isnt used much, and i would use the AMD card for the passthrough.

@idkthegithub here is the guide: https://github.com/foxlet/macOS-Simple-KVM/blob/master/docs/guide-passthrough.md
